media/server/main/source/MediaKeysCapabilities.cpp

```diff
@@ -29,6 +29,8 @@ namespace
 /**
  * @brief Coverts MediaKeyErrorStatus to string.
  */
+
+#ifdef RIALTO_LOG_ERROR_ENABLED
```

Comment:
why do we need it to be compiled only when error log is enabled?

Reply:
so are you going to remove this ifdef or it's going to stay?

Reply:
If we're keeping the FATAL, ERROR, WARN and MIL enabled for release mode, then I'll remove it